vocabularies/DirectEdit.xml

```diff
@@ -0,0 +1,79 @@
+        <String>2020-07-10 © Copyright 2020 SAP SE. All rights reserved</String>
+      </Annotation>
+      <Annotation Term="Core.Links">
+        <Collection>
+          <Record>
+            <PropertyValue Property="rel" String="latest-version" />
+            <PropertyValue Property="href" String="https://sap.github.io/odata-vocabularies/vocabularies/DirectEdit.xml" />
+          </Record>
+          <Record>
+            <PropertyValue Property="rel" String="alternate" />
+            <PropertyValue Property="href" String="https://sap.github.io/odata-vocabularies/vocabularies/DirectEdit.json" />
+          </Record>
+          <Record>
+            <PropertyValue Property="rel" String="describedby" />
+            <PropertyValue Property="href" String="https://github.com/sap/odata-vocabularies/blob/master/vocabularies/DirectEdit.md" />
+          </Record>
+        </Collection>
+      </Annotation>
+
+      <Term Name="SideEffects" Type="DirectEdit.SideEffectsType" Nullable="false" AppliesTo="EntitySet">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="Determine side effects of client-side data modification" />
+      </Term>
+
+      <ComplexType Name="SideEffectsType">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.LongDescription">
+          <String>After a change to a property whose path is contained in `Triggers`, the client should pass the entity with the changed property to the `CalculationFunction` and receive the entity with all changes that happen as side effects of the given change.</String>
+        </Annotation>
+        <Property Name="Triggers" Type="Collection(Edm.String)" Nullable="false">
+          <Annotation Term="Core.Description" String="List of paths to the properties whose changes should trigger the side effects calculation." />
+          <Annotation Term="Validation.MinItems" Int="1" />
+        </Property>
+        <Property Name="CalculationFunction" Type="Common.QualifiedName" Nullable="false">
+          <Annotation Term="Core.Description" String="The operation calculating the side effects. While non-changing for the service, this technically is an action bound to the entity set so that the parameters can be sent in the POST request body. The action has the following non-binding parameters:
+
+- `Qualifier` of type [`Core.SimpleIdentifier`](https://github.com/oasis-tcs/odata-vocabularies/blob/master/vocabularies/Org.OData.Core.V1.md#SimpleIdentifier)
+    or cast-compatible: the qualifier of the `SideEffects` annotation
+
+- `Trigger` of type `Edm.String`: the trigger of side-effects determination, see `Triggers` property
+
+- `Data` of either the entity type of the annotated entity set or a complex type that is structure-compatible with it
+
+The return type of the action also needs to be either the entity type of the annotated entity set or structure-compatible with it,
+it can be the same type as for `Data`.
```

Comment:
In the md file this is right in the middle of the action description since the description of 'structure compatible' is right behind it.
Is there a way to mark 'return type' more obviously so that you can immediately detect where the return signature starts (the 3 input parameters are clearly visible)? Would it make sense to move this to the end (after defining 'structure compatible') or could the definition of 'structure compatible' be separated/indented somehow to show that this is just additional information?

Reply:
```
You can use **bold** or *italic*.
```
You can use **bold** or *italic*.

Reply:
```suggestion
The **return type** of the action also needs to be either the entity type of the annotated entity set or structure-compatible with it,
it can be the same type as for `Data`.
```